tests/integrations/test_db_integrations_sql.py

```diff
@@ -206,12 +241,68 @@ def _get_sql_span_for_request(self, weblog_request):
                         )
                         return span_child
 
 
-class Test_Postgres_db_integration(_BaseIntegrationsSqlTestClass):
+@scenarios.integrations
+class _BaseAgentIntegrationsSqlTestClass(_BaseIntegrationsSqlTestClass):
+    def test_sql_query(self):
+        """ Usually the query """
+        for db_operation, request in self.requests[self.db_service].items():
+            if db_operation not in ["procedure", "select_error"]:
+                span = self._get_sql_span_for_request(request)
+                assert db_operation in span["meta"]["sql.query"].lower()
+
+    def test_obfuscate_query(self):
+        """ All queries come out obfuscated from agent """
+        for db_operation, request in self.requests[self.db_service].items():
+            span = self._get_sql_span_for_request(request)
+            # We launch all queries with two parameters (from weblog)
+            # Insert and procedure:These operations also receive two parameters, but are obfuscated as only one.
+            if db_operation in ["insert", "procedure"]:
+                assert span["meta"]["sql.query"].count("?") == 1
+            else:
+                assert span["meta"]["sql.query"].count("?") == 2
+
+    def _get_sql_span_for_request(self, weblog_request):
+        for data, span in interfaces.agent.get_spans(weblog_request):
+            logger.info(
+                f"Agent: Span found with trace id: {span['traceID']} and span id: {span['spanID']} in file {data['log_filename']}"
+            )
+            content = data["request"]["content"]["tracerPayloads"]
+            for payload in content:
+                for chunk in payload["chunks"]:
+                    for span_child in chunk["spans"]:
+                        if (
+                            "type" in span_child
+                            and span_child["type"] == "sql"
+                            and span_child["traceID"] == span["traceID"]
+                            and span_child["resource"]
+                            != "SELECT ?"  # workaround to avoid conflicts on connection check on mssql
+                        ):
+                            logger.debug("Agent: Span type sql found!")
+                            logger.info(
+                                f"Agent: Span SQL found with trace id: {span_child['traceID']} and span id: {span_child['spanID']}"
+                            )
+                            logger.debug(f"Agent: Span: {span_child}")
+                            return span_child
+
+
+############################################################
+# Postgres: Tracer and Agent validations
+############################################################
+class Test_Tracer_Postgres_db_integration(_BaseTracerIntegrationsSqlTestClass):
+    db_service = "postgresql"
+
+    @missing_feature(library="python", reason="Python is using the correct span: db.system")
```

Comment:
if all tests are failing on a tracer, it's better to store this info in manifest file. (Same comment for other class in the same case)

Reply:
I added in the manifest for dotnet, golang, php....